en/drafts/originals/agent-based-model-communication-networks.md

```diff
@@ -474,7 +478,7 @@ class LetterAgent(mesa.Agent):
         self.model.grid.move_agent(self, new_position)
 ```
 
-Next, we need to get all the other agents present in a cell, and send one of them a letter. We can get the contents of one or more cells using the grid's `get_cell_list_contents` method, or by accessing a cell directly. The method accepts a list of cell coordinate tuples, or a single tuple if we only care about one cell.
+Next, we need to find all the other agents present in our agent's new cell, and send one of them a letter. We can get the contents of one or more cells using the grid's `get_cell_list_contents` method, or by accessing a cell directly. The method accepts either a single coordinate tuple, or a list of tuples.
```

Comment:
What do you mean by 'accessing a cell directly'?

Reply:
@maltevogl what method did you mean exactly? I'm not sure

Reply:
Cells can also be accessed by an index. So we could extend the text by : "or by accessing a cell directly by its internal number."

Reply:
Sorry @maltevogl, I'm still a little lost. When you say "accessing a cell", do you mean retrieving a list of the cell's contents? By "internal number", do you mean the cell coordinates? 

In the sentence that comes just after, is "The method" the first method (`get_cell_list_contents`) or the second method (accessing a cell)?

Reply:
No problem at all. I think it is not really necessary to keep this part of the sentence if it leads to confusion. The get_cell_list_contents method is meant to be used. So if you are fine with it, I would suggest to remove the part after the comma: ", or by accessing a cell directly." Would that be OK? That would also solve the problem with the next sentence, since there is then only one method.

Reply:
Thanks @maltevogl, I think that's a good solution. It's clearer just to focus on one method!